test/csuite/timestamp_abort/main.c

```diff
@@ -438,18 +444,18 @@ run_workload(uint32_t nth)
     td = dcalloc(nth + 2, sizeof(THREAD_DATA));
 
     /*
-     * Size the cache appropriately for the number of threads. Each thread generally adds keys
-     * sequentially to its own portion of the key space, so each thread will be dirtying one page at
-     * a time. By default, a leaf page grows to 32K in size before it splits and the thread begins
-     * to fill another page. We'll budget for 5 full size leaf pages per thread in the cache plus a
-     * little extra in the total for overhead.
+     * Size the cache appropriately for the number of threads. Each thread adds keys sequentially to
+     * its own portion of the key space, so each thread will be dirtying one page at a time. By
+     * default, a leaf page grows to 32K in size before it splits and the thread begins to fill
+     * another page. We'll budget for 10 full size leaf pages per thread in the cache plus a little
+     * extra in the total for overhead.
      *
      * The configuration sets the eviction update and dirty targets at 20% so that on average, each
-     * thread can have a dirty page before eviction threads kick in. On the other side, the eviction
-     * update and dirty triggers are 95%, so application threads aren't involved in eviction until
-     * we're close to running out of cache.
+     * thread can have a couple of dirty pages before eviction threads kick in. On the other side,
+     * the eviction update and dirty triggers are 90%, so application threads aren't involved in
```

Comment:
I have a minor concern that having the `90%` here in the comment but the value way up at the top of the file in the configuration string means they will get out of sync with each other.

Reply:
I've restructured the comments a bit.